_episodes/02-filedir.md

```diff
@@ -200,11 +200,19 @@ to use a command and what options it accepts ---
 
 We'll describe both ways next.
 
+> ## Help for built-in commands
+>
+> Some commands are built in to the Bash shell, rather than existing as separate
+> programs on the filesystem. One example is the `cd` (change directory) command.
+> If you get a message like `No manual entry for cd`, try `help cd` instead. The
+> `help` command is how you get usage information for
+> [Bash built-ins](https://www.gnu.org/software/bash/manual/html_node/Bash-Builtins.html).
+{: .callout}
+
 #### The `--help` option
 
-Most bash commands and programs that people have written to be
-run from within bash, support a `--help` option that displays more
-information on how to use the command or program.
+Many terminal-based programs recognize a `--help` option that displays more detailed
+information about how to use it.
```

Comment:
Would leave this as is, there are other shells, and this can be mentioned, but `terminal-based` program is too vague. 
```suggestion
Most bash commands and programs that people have written to be
run from within bash, support a `--help` option that displays more
information on how to use the command or program.
```

Reply:
Consistently using capital B is good.  The lesson has been written for Bash shell, but the concepts are similar for other shells.  Testing with Z shell would be good, a pull request to update setup for Mac once this is done would be much appreciated.

Reply:
Can also update the lesson to ensure all commands work on Bash and Zsh, but clarify that there are cases where these differ.